cdk/lib/newsletters-tool.ts

```diff
@@ -56,6 +66,43 @@ export class NewslettersTool extends GuStack {
 	private setUpNodeEc2 = (props: NewslettersToolProps) => {
 		const { app, domainName } = props;
 
+		// To avoid exposing the bucket name publicly, fetches the bucket name from SSM (parameter store).
+		const bucketSSMParameterName = `/${this.stage}/${this.stack}/newsletters-api/s3BucketName`;
+		const bucketName = StringParameter.valueForStringParameter(
+			this,
+			bucketSSMParameterName,
+		);
+		const dataStorageBucket = new GuS3Bucket(this, 'DataBucket', {
+			bucketName,
+			app: props.app,
+			versioned: true,
+		});
+
+		const s3AccessPolicy = new GuPolicy(this, `${app}-InstancePolicy`, {
+			policyName: 'root',
```

Comment:
give appropriate name

Reply:
Would be good to update the policyName before this actually gets used